alert-common/src/main/java/com/synopsys/integration/alert/common/message/model/LinkableItem.java

```diff
@@ -98,7 +98,7 @@ public int compareTo(final LinkableItem otherItem) {
 
     @Override
     public String toString() {
-        ReflectionToStringBuilder builder = new ReflectionToStringBuilder(this, RecursiveToStringStyle.JSON_STYLE);
+        ReflectionToStringBuilder builder = new ReflectionToStringBuilder(this, ToStringStyle.JSON_STYLE.JSON_STYLE);
```

Comment:
I'm pretty sure we use the RecursiveToStringStyle in our libraries. What was this issue with it?

Reply:
The sonar cloud issue was that we were using a child class to reference a static in a parent class.  The fix is to use the parent class in the static reference.